src/qt/forms/debugwindow.ui

```diff
@@ -1592,6 +1592,84 @@
                 </widget>
                </item>
                <item row="23" column="0">
+                <widget class="QLabel" name="peerAddrRelayEnabledLabel">
+                 <property name="toolTip">
+                  <string extracomment="Tooltip text for the Address Relay field in the peer details area.">Whether we relay addresses to this peer.</string>
+                 </property>
+                 <property name="text">
+                  <string>Address Relay</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="23" column="1">
+                <widget class="QLabel" name="peerAddrRelayEnabled">
+                 <property name="cursor">
+                  <cursorShape>IBeamCursor</cursorShape>
+                 </property>
+                 <property name="text">
+                  <string>N/A</string>
+                 </property>
+                 <property name="textFormat">
+                  <enum>Qt::PlainText</enum>
+                 </property>
+                 <property name="textInteractionFlags">
+                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
+                 </property>
+                </widget>
+               </item>
+               <item row="24" column="0">
+                <widget class="QLabel" name="peerAddrProcessedLabel">
+                 <property name="toolTip">
+                  <string extracomment="Tooltip text for the Addresses Processed field in the peer details area.">Total number of addresses processed, excluding those dropped due to rate-limiting.</string>
+                 </property>
+                 <property name="text">
+                  <string>Addresses Processed</string>
```

Comment:
A [feedback](https://www.transifex.com/bitcoin/bitcoin/translate/#fr/qt-translation-023x/410183859) from Transifex translator:
> Rather: "Processed Addresses"

Reply:
If "processed" was an adjective, then I would agree with the feedback, i.e. "red car" and not "car red".

However, in this case it is a verb and this is fine as-is, i.e. "cars sold" vs "sold cars".

Reply:
Hi @jonatack The lack of string context in Transifex is a real problem for this project, as proper context (dev notes and/or screenshots) are essential to achieve quality translations.
How is this string used, is this a title, a tooltip, a notification? Do we actually mean “Addresses are processed”?

Reply:
@AO-LocLab it is the field name in the peer details area, as displayed in the screenshot in the PR description. There is also a tooltip saying this, and its explanatory description, 3 lines higher at line 1623. I don't know if these are visible in transifex.

```html
<string extracomment="Tooltip text for the Addresses Processed field in the peer details area.">
    Total number of addresses processed, excluding those dropped due to rate-limiting.</string>
```

Reply:
> I don't know if they are visible in transifex.

![Screenshot from 2022-02-20 19-09-24](https://user-images.githubusercontent.com/32963518/154854876-1f3491d2-b056-47e8-aa75-70e66d750238.png)

Reply:
Although, this one has no developer notes:

![Screenshot from 2022-02-20 19-12-06](https://user-images.githubusercontent.com/32963518/154855042-3a7155fe-cc2a-4b4e-8d2c-9df658d44f39.png)

Reply:
> Although, this one has no developer notes:

Should an extracomment be added, or something else?

Reply:
> > Although, this one has no developer notes:
> 
> Should an extracomment be added, or something else?

Actually, `extracomment`s in `*.ui` files _are_ Qt [translator comments](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) which are presented on Transifex as "developer notes".

Reply:
> Should an extracomment be added, or something else?

That is the plan :)

Reply:
Ok! Will propose adding extracomments to the titles, then (if I understand correctly) and maybe to the tooltips without them.

Reply:
> Ok! Will propose adding extracomments to the titles, then (if I understand correctly) and maybe to the tooltips without them.

Thank you!

Reply:
Started adding and improving the dev notes (and tooltips) in #554.